src/cmd/longevity_test/longevity.go

```diff
@@ -16,20 +16,25 @@ import (
 	"github.com/alcionai/corso/src/pkg/logger"
 	"github.com/alcionai/corso/src/pkg/path"
 	"github.com/alcionai/corso/src/pkg/repository"
+	"github.com/alcionai/corso/src/pkg/storage"
 	"github.com/alcionai/corso/src/pkg/store"
 )
 
 // deleteBackups connects to the repository and deletes all backups for
 // service that are at least deletionDays old. Returns the IDs of all backups
 // that were deleted.
+// Only supported for S3 repos currently.
```

Comment:
To extend support to non-s3 storage providers, we'd need to pass in new cli flags to longevity tests to indicate provider type.

Reply:
Will there be a PR coming for this support?  Seems like the change will be incomplete without it.

Reply:
this is the longevity test command that is only used in longevity testing. I think for the moment at least we can punt adding support for this until a bit later. The commands like this are already a bit hacky because they hook into some parts of the CLI config parsing but not all of it. TBH I think refactoring the config parsing code so it's more portable would provide the most benefit towards bringing CI commands like this into a better state